### PR TITLE
#1 create chargify package

### PR DESCRIPTION
**Pull request by agreggio-slingr**
Pull-request for issue #[14457](https://github.com/slingr-stack/platform/issues/14457) created by @agreggio-slingr 

## What it does?

Create the Chargify package

## How to test it?

You can install the following test package with the following credentials.

ssh: git@github.com:slingr-stack/chargify-package-test.git

```json
{
"subdomain":"slingr-local-tests",
"apiKey":"bUEq9IPFH9tPGYEgA8fV11yLa0qxhI1o0UEG5qaKA",
"siteSharedKey":"LmvJ4q0ZbJnCzi3sVtBaUf600RE4wgNcaBjb74ibTs"
}
```

To easily test, you can use the console

pkg.chargify.functions.coupons.findJson.get();

## Implementation notes

N/A

### CODE REVIEW

**Review by jguardiolagarcia — APPROVED** [review]

File: listeners/listeners.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+/****************************************************
+ Listeners
+ ****************************************************/
+
+listeners.defaultWebhook = {
```
Comment (pasaperez-slingr) [inline listeners/listeners.js:5]: @agreggio-slingr can you change this listener name?

File: scripts/functions.js [hunk 2]
```diff
@@ -0,0 +1,3147 @@
+/****************************************************
+ Dependencies
+ ****************************************************/
+
+var httpService = svc.http;
```
Comment (pasaperez-slingr) [inline scripts/functions.js:5]: @agreggio-slingr Here you use the http not as a dependency, make sure to use the latest from the helper or the other packages.

File: listeners/listeners.js (hunk 1 above)
Comment (agreggio-slingr) [reply listeners/listeners.js:5]: Yes, I can

File: README.md [hunk 3]
```diff
@@ -0,0 +1,1737 @@
...
+The Javascript API of the chargify endpoint has three pieces:
+
+- **HTTP requests**: These allow to make regular HTTP requests.
+- **Shortcuts**: These are helpers to make HTTP request to the API in a more convenient way.
+- **Additional Helpers**: These helpers provide additional features that facilitate or improves the endpoint usage in SLINGR.
+
+## HTTP requests
+You can make `GET`,`POST`,`DELETE`,`PUT`,`PATCH` requests to the [chargify API](API_URL_HERE) like this:
+```javascript
+var response = app.endpoints.chargify.get('/transactions/:transaction_id.json')
```
Comment (pasaperez-slingr) [inline README.md:56]: @agreggio-slingr mentions to app.endpoints you can reapply de helper generator, this fix all the issues.

**Comment by pasaperez-slingr** [comment]
Add LICENSE icon48.png files

**Pull request merged**